platform/ios/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
+## Master
+
+* Add a `maximumWidthRatio` property for customizing maximum width of scale bar. ([#65](https://github.com/mapbox/mapbox-gl-native-ios/pull/65))
```

Comment:
The blurb in changelogs should start with a verb in past tense. When you add a property you have to include the class name in order to link the documentation.
Please change the above to:
> Added \`MGLMapView.maximumWidthRatio\` property for customizing the width ratio of map view's scale bar.